internal/tui/templates/templater.go

```diff
@@ -27,6 +29,22 @@ func SetCustomUsageFunc(cmd *cobra.Command) error {
 	return nil
 }
 
+// getTerminalWidth returns the width of the terminal, defaulting to 80 if it cannot be determined
+func getTerminalWidth() int {
+	defaultWidth := 80
+	screenWidth := defaultWidth
+
+	// Detect terminal width and use it by default if available
+	if term.IsTerminal(int(os.Stdout.Fd())) {
+		termWidth, _, err := term.GetSize(int(os.Stdout.Fd()))
+		if err == nil && termWidth > 0 {
+			screenWidth = termWidth - 2
```

Comment:
Consider documenting the padding constant and adding bounds checking.

Reply:
https://github.com/cloudposse/atmos/pull/815#pullrequestreview-2473474373

Reply:
This is resolved